components/inputs/docs/input-date-time.md

```diff
@@ -112,82 +112,110 @@ To make your usage of `d2l-input-date-time` accessible, use the following proper
-const localDateTime = convertUTCToLocalDateTime(UTCDateTimeObject);
-const value = formatDateInISO(localDateTime); // use as value in input-date
+const localTime = getLocalTimeFromUTCDateTime(UTCDateTime); // for later conversion back to UTC
+let value = getLocalDateFromUTCDateTime(UTCDateTime); // use as value in input-date
+
+// ...input-date usage code...
+
+value = '12:20:30'; // value output from input-date
+const isoFormattedUTCDateTime = getUTCDateTimeFromLocalDateTime(value, localTime);
 ```
-### Convert UTC Date/Time to Local Time
 
-To convert a UTC date/time (presumably in ISO 8601 format, `YYYY-MM-DDTHH:mm:ss.sssZ`, adjust if not) to a time in the user's timezone for use in `input-time`:
-- Parse into object (`parseISODateTime` from `helpers/dateTime.js`)
-- Convert UTC object into local timezone object (`convertUTCToLocalDateTime` from the [intl library](https://github.com/BrightspaceUI/intl#datetime-conversion-based-on-user-timezone))
-- Format as ISO 8601 date or time (`formatTimeInISO` from `helpers/dateTime.js`)
+Example of usage in component (simplified LitElement):
 
 ```javascript
-import {convertUTCToLocalDateTime} from '@brightspace-ui/intl/lib/dateTime.js';
-import {formatTimeInISO, parseISODateTime} from '@brightspace-ui/core/helpers/dateTime.js';
-
-const UTCDateTime = '2018-03-01T12:20:00.000Z';
-const UTCDateTimeObject = parseISODateTime(UTCDateTime);
-const localDateTime = convertUTCToLocalDateTime(UTCDateTimeObject);
-const value = formatTimeInISO(localDateTime); // use as value in input-time
+import '@brightspace-ui/core/components/inputs/input-date.js';
+import {getLocalDateFromUTCDateTime, getLocalTimeFromUTCDateTime, getUTCDateTimeFromLocalDateTime} from '@brightspace-ui/core/helpers/dateTime.js';
+
+class MyElement extends LitElement {
+	static get properties() {
+		return {
+			utcDateTime: { type: String, attribute: 'utc-date-time' } // e.g., '2018-03-01T12:20:00.000Z'
+			_value: { type: String }
+		};
+	}
+
+	firstUpdated(changedProperties) {
+		super.firstUpdated(changedProperties);
+
+		this._value = getLocalDateFromUTCDateTime(UTCDateTime);
+		this._initialLocalTime = getLocalTimeFromUTCDateTime(UTCDateTime);
+	}
+
+	render() {
+		return html`
+			<d2l-input-date @change="${this._handleChange}" value="${this._value}"></d2l-input-date>
+		`;
+	}
+
+	_handleChange(e) {
+		const value = e.target.value;
+		const isoFormattedUTCDateTime = getUTCDateTimeFromLocalDateTime(value, this._initialLocalTime);
+	}
+}
+customElements.define('d2l-my-element', MyElement);
 ```
 
-### Convert Local Date to UTC Date/Time
+### input-time: Convert UTC Date/Time to Local Timezone and Back to UTC
+
+To convert a UTC date/time (presumably in ISO 8601 format, `YYYY-MM-DDTHH:mm:ss.sssZ`, adjust if not) to a time in the user's timezone for use in `input-time`, use `getLocalTimeFromUTCDateTime` from `helpers/dateTime.js`, which will return the time in the user's timezone in the ISO 8601 date format `hh:mm:ss`.
```

Comment:
Same comment here I think. I could be wrong, but I think in most cases where folks are dealing with only a time, they'll just have the `HH:mm:ss.sssZ` bit and even if they have the date part they don't care about preserving it.